### PR TITLE
Improve memeolist README

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

File: memeolist/README.md [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
+
+To create a meme the app will upload the selected image from the app to [imgur.com](http://imgur.com), generate a meme (and a public link) using [memegen](https://memegen.link) and store it on the [AeroGear Data Sync Server](https://github.com/aerogear/data-sync-server).
+
+## Version matrix
+
+| memeolist | Android SDK | AeroGear Data Sync Server  |
+|:---------:|:-----------:|:--------------------------:|
+| [2.1.0-alpha.1](https://github.com/aerogear/aerogear-android-example-apps/releases/tag/2.1.0-alpha.1) | [2.1.0-alpha.1](https://github.com/aerogear/aerogear-android-sdk/releases/tag/2.1.0-alpha.1) | [0.1.0-alpha](https://github.com/aerogear/data-sync-server/releases/tag/0.1.0-alpha) |
+
+:point_right: If you are brave and wanna live in the edge you can try it from the master branch, but you will need to build the [AeroGear Android SDK](https://github.com/aerogear/aerogear-android-sdk) snapshot dependency yourself, but keep in mind master branch is unstable so use it at your own risk.
```
Comment (wtrocki) [inline memeolist/README.md]: I will rephrase this to - master branch contain development changes that may not work with the released server. 
Please use master only if you are contributing to the application.

Comment (danielpassos) [reply memeolist/README.md]: @wtrocki wdyt about:

:point_right: master branch contains development changes that may not work with the released server. If you wanna play with the master branch you will need to build the [AeroGear Android SDK](https://github.com/aerogear/aerogear-android-sdk) snapshot dependency yourself. Please use master only if you are contributing to the application.

Comment (wtrocki) [reply memeolist/README.md]: This is exactly what @darahayes was looking for! Great change!

**Comment by wtrocki** [comment]
@danielpassos I will wait for @darahayes opinion, but this sounds very reasonable. 
This is actually an good base for building getting started on mobile-docs.
